qacode/core/webs/pages/PageLogin.py

```diff
@@ -0,0 +1,28 @@
+from qacode.core.webs.pages.PageBase import PageBase
+from qacode.core.exceptions.PageException import PageException
+
+
+class PageLogin(PageBase):
+    """Inherit PageBase class with login and valitation methods"""
+
+    txt_username = None
+    txt_password = None
+    btn_login = None
+
+    def __init__(self,bot, url, by=By.CSS_SELECTOR, selectors=[], go_url=True):
+        super().__init__(bot, url, by ,selectors, go_url)
+        if selectors is None or len(selectors) != 3:
+            raise PageException("Can't generate PageLogin elements if not have 3 selectors")
+        self.txt_username = self.elements[0]
+        self.txt_password = self.elements[1]
+        self.btn_login = self.elements[2]    
+
+    def login(self,username,password,is_login_now=True):
+        message_error = "Can't '{}' '{}' on field '{}' for class PageLogin"
+        try: self.txt_username.send_keys(username)
```

Comment:
Compound statements (multiple statements on one line, like this `try`) are discouraged because it hurts readability.

Please read the last point of https://www.python.org/dev/peps/pep-0008/#other-recommendations.

Reply:
just temporal, ControlBase must be FIX this #47